Tasks/merge.js

```diff
@@ -1,19 +1,7 @@
+'use strict';
 // Merge two dictionaries
 
-merge_two_objects=(function merge_two_objects(object_1,object_2){
-    object_3 = [({})][0];
-   for (attribute_name in object_1) {
-       (
-          object_3[attribute_name] == object_1[attribute_name]
-       );
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const merge = (obj1, obj2) => ({ ...obj1, ...obj2 });
```

Comment:
You can add 1-line version in comments but plz roll back to loops

Reply:
fixed